packages/Telephony/Actions/StartRecording.cs

```diff
@@ -38,12 +37,6 @@ public StartRecording([CallerFilePath] string sourceFilePath = "", [CallerLineNu
             this.RegisterSourceLocation(sourceFilePath, sourceLineNumber);
 
             this.CommandName = RecordingStart;
-
-            this.Data = new RecordingStartSettings()
```

Comment:
Is this not relevant anymore?

Reply:
yeah this was removed because we only support a single value for the settings right now. This will come back as we support more settings.